fixfasta.py

```
@@ -0,0 +1,446 @@
+    logging.basicConfig(
+        level=log_level,
+        format="[%(levelname)s] %(message)s",
+        stream=sys.stderr
+    )
+    
+    try:
+        # Setup input/output
+        if args.files:
+            input_stream = fileinput.input(args.files)
+        else:
+            input_stream = sys.stdin
```

Ruff (0.11.9)

Use ternary operator `input_stream = fileinput.input(args.files) if args.files else sys.stdin` instead of `if`-`else`-block

Replace `if`-`else`-block with `input_stream = fileinput.input(args.files) if args.files else sys.stdin`

(SIM108)